src/js/view/view.js

```diff
@@ -351,6 +351,12 @@ define([
                 // area around scrollbar on the playlist.  skin fix required to remove
                 '.jw-playlist-container ::-webkit-scrollbar'
             ], 'border-color', backgroundColor);
+
+            // Set global colors, used by related plugin
+            utils.css('#' + id + ' .jw-color-active', { color: activeColor, borderColor: activeColor, stroke: activeColor }, id);
+            utils.css('#' + id + ' .jw-color-active-hover:hover', { color: activeColor, borderColor: activeColor, stroke: activeColor }, id);
+            utils.css('#' + id + ' .jw-color-inactive', { color: inactiveColor, borderColor: inactiveColor }, id);
+            utils.css('#' + id + ' .jw-color-inactive-hover:hover', { color: inactiveColor, borderColor: inactiveColor, stroke: inactiveColor }, id);
```

Comment:
I didn't want to use addStyle since it accepts only one property per selector, and I need to put three per. `addStyles` calls `utils.css` anyway. Only thing I'm missing is a check for null styles